config/initializers/spree.rb

```diff
@@ -12,8 +12,6 @@
 require "#{Rails.root}/app/models/spree/payment_method_decorator"
 require "#{Rails.root}/app/models/spree/gateway_decorator"
 
-Spree::Api::Config[:requires_authentication] = true
-
```

Comment:
@sauloperez can you give a little explanation of what this is doing?

Reply:
Sure! this is the exact line that was forcing the api_key authentication. Without it we rely on the session auth instead. This is what I implemented in https://github.com/coopdevs/spree/commit/4e658f5f08f1a0e5e315cae9fdc52d08b1708082.

Does that make sense?